bin/run-fisher.sh

```diff
@@ -5,6 +5,8 @@ if [ ! -f DeepSpeech.py ]; then
     exit 1
 fi;
 
+XDG_DATA_HOME=${XDG_DATA_HOME:="$HOME/.local/share"}
```

Comment:
This is already what the `pyxdg` module does: https://cgit.freedesktop.org/xdg/pyxdg/tree/xdg/BaseDirectory.py#n31 :)